packages/email-notification-dispatcher/src/handlers/agreements/handleAgreementEvent.ts

```diff
@@ -70,20 +74,60 @@ export async function handleAgreementEvent(
         correlationId,
       })
     )
+    .with({ type: "AgreementSuspendedByProducer" }, ({ data: { agreement } }) =>
+      handleAgreementSuspendedByProducerToConsumer({
+        agreementV2Msg: agreement,
+        logger,
+        readModelService,
+        templateService,
+        userService,
+        correlationId,
+      })
+    )
+    .with({ type: "AgreementSuspendedByPlatform" }, ({ data: { agreement } }) =>
+      handleAgreementSuspendedByPlatformToConsumer({
```

Comment:
in this case (and for unsuspended), the notification should be both to consumer and to producer: so I suppose, after a merge or rebase, this should be edited to be a concat of these notifications plus those produced by the handler in #2268?